GooglePlacesAutocomplete.d.ts

```diff
@@ -365,12 +370,29 @@ interface GooglePlacesAutocompleteProps extends TextInputProps {
   renderRightButton?: React.ComponentType<{}>;
 
   // sets the request URL to something other than the google api.  Helpful if you want web support or to use your own api.
-  requestUrl?: {
-    url: string;
-    useOnPlatform: 'web' | 'all';
-  };
+  requestUrl?: RequestUrl;
+}
+
+interface InitialState {
+  text: string;
+  dataSource: any[];
+  listViewDisplayed: boolean;
+  url: string;
 }
 
 export class GooglePlacesAutocomplete extends React.Component<
   GooglePlacesAutocompleteProps
-> {}
+> {
```

Comment:
same as above. These are never exposed to the component.

Reply:
some of these can be used externally (with a ref for example).  

Only `triggerFocus` and `triggerBlur` are public methods.

Reply:
As I said above, in my understanding, public methods should not have the prefix underlined in front of the method names, so I added these methods

Reply:
the declarations of these methods will not be maintained?

Reply:
They are internal